### PR TITLE
 Fix(web-twig): Deprecate `class` prop in the `Header` abstract components

### PR DESCRIPTION
**Pull request by dlouhak**
## Description

[I walked around](https://github.com/lmc-eu/spirit-design-system/pull/990/files#diff-c3ca80c4a6e6d51e987124ce945d2fcb8ad6523f1521ae65babf9a68504e763e). 🚶‍♀️

### CODE REVIEW

**Review by adamkudrna — COMMENTED** [review]
Thank you for fixing this! 🙏🏻 But, since it's a BC, we cannot merge it right now.

Are you able to turn it into a backwards-compatible change? 🤔

**Comment by literat** [comment]
> Thank you for fixing this! 🙏🏻 But, since it's a BC, we cannot merge it right now.
> 
> Are you able to turn it into a backwards-compatible change? 🤔

Yeah, we should do the deprecation here anyway. @dlouhak Do you need assistance?

**Comment by dlouhak** [comment]
> > Thank you for fixing this! 🙏🏻 But, since it's a BC, we cannot merge it right now.
> > Are you able to turn it into a backwards-compatible change? 🤔
> 
> Yeah, we should do the deprecation here anyway. @dlouhak Do you need assistance?

Mission completed. ✅

File: packages/web-twig/src/Resources/components/Header/_abstracts/Button.twig [hunk 1]
```diff
@@ -7,10 +7,23 @@
...
-{%- set _classNames = [ _rootClassName, _class ] -%}
+{## `_propsWithoutClassAndStyleProps` is a temporary prop until the `class` prop will be removed ##}
+{%- set _propsWithoutClassAndStyleProps = props | filter((value, prop) => prop is not same as('class') and prop is not same as('style')) -%}
+
+{%- set _styleProps = useStyleProps(_propsWithoutClassAndStyleProps) -%}
+{%- set _classNames = [ _rootClassName, _class, _styleProps.className] -%}
+
+{# Deprecations #}
+{% if _class %}
+    {% deprecated 'Header/_abstracts/Button: The "class" property is deprecated, use "UNSAFE_className" instead.' %}
```
Comment (literat) [inline packages/web-twig/src/Resources/components/Header/_abstracts/Button.twig:18]:
```suggestion
    {% deprecated 'Header/_abstracts/Button: The "class" property is deprecated and will be removed in the next major version, use "UNSAFE_className" instead.' %}
```

**Review by literat — APPROVED** [review]
LGTM 👍 Just do a better copy of the deprecation message as suggested.

**Pull request merged**